static/scss/post.scss

```diff
@@ -52,6 +52,16 @@
     }
   }
 
+  tt, code, kbd, samp {
```

Comment:
Insert this comment to fix failing tests:
Add this line here to fix failing tests:
```javascript
        // sass-lint:disable no-vendor-prefixes
```